homeworks/08-cats-catcher/src/App/generateFiled.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect, useCallback } from "react";
+import random from "lodash/random";
+
+import { WINDOW_SIZE } from "./contants";
+
+const generateGameField = (): boolean[][] =>
+  Array.from({ length: 10 }, () => Array.from({ length: 10 }, () => true));
+
+const getPositions = (field: boolean[][]): number[] => {
+  const x = random(0, 9);
+  const y = random(0, 9);
+
+  const isCellFree = field[x][y];
+
+  if (isCellFree) {
+    field[x][y] = false;
```

Comment:
Don't mutate input arguments, especially in `get` functions! 
It is not obvious when you call `const positions = getPositions(gameFiled)` and as a side-effect of this calling will be changing `gameField` variable.